packages/next/build/webpack-config.ts

```diff
@@ -393,32 +393,36 @@ export default async function getBaseWebpackConfig(
               hash.update(module.libIdent({ context: dir }))
             }
 
-            return hash.digest('hex').substring(0, 8)
+            return isWebpack5
+              ? 'static/chunks/'
+              : '' + hash.digest('hex').substring(0, 8)
           },
           priority: 30,
           minChunks: 1,
           reuseExistingChunk: true,
         },
         commons: {
-          name: 'commons',
+          name: (isWebpack5 ? 'static/chunks/' : '') + 'commons',
           minChunks: totalPages,
           priority: 20,
         },
         shared: {
           name(module, chunks) {
-            return (
-              crypto
-                .createHash('sha1')
-                .update(
-                  chunks.reduce(
-                    (acc: string, chunk: webpack.compilation.Chunk) => {
-                      return acc + chunk.name
-                    },
-                    ''
-                  )
-                )
-                .digest('hex') + (isModuleCSS(module) ? '_CSS' : '')
-            )
+            return isWebpack5
+              ? 'static/chunks/'
+              : '' +
```

Comment:
This looks like a bug, did you mean to add parentheses?

```suggestion
            return (isWebpack5
              ? 'static/chunks/'
              : '') +
```

Reply:
I did!